tests/KAT/build_kats.sh

```diff
@@ -1,13 +1,8 @@
 #!/bin/bash
 set -e
 
-current_dir="${PWD##*/}"
-if [ $current_dir != "KAT" ];
-then
-  echo "Script needs to be run from inside the KAT folder";
-  echo "Current working directory: "$current_dir
-  exit;
-fi;
```

Comment:
Sidequest from the main point of the PR. But this makes this script a bit more flexible. Not needing to run it from a certain location.

Reply:
Just to be clear, on a fresh system you have run this from a different random directory, then run the KAT vectors and it works? I haven't checked it yet.

The test shell scripts are somewhat pushing the level I am already ok with before re-writing it in something like Python, they already have a lot of complexity that isn't easy to grok at first glance, am trying to avoid that in the entire codebase including the tests.

Reply:
> Just to be clear, on a fresh system you have run this from a different random directory, then run the KAT vectors and it works? I haven't checked it yet.

Yeah, the change to the script `cd` into the script dir. So the end result is the same. It's just that the script will `cd` for you instead of giving you an annoying error telling you to `cd` for it :)